openlibrary/templates/login.html

```diff
@@ -135,7 +135,7 @@
         </div>
         <div class="formElement">
             <div class="label"></div>
-            <div class="input smaller">$:_('<a href="/account/password/forgot">Forgot Password</a>?')
+            <div class="input smaller">$:_('<a href="https://archive.org/account/login.forgotpw.php">Forgot Archive.org Password</a>?')
```

Comment:
Prior to August 1, should this "Forgot Password" link indeed be limited to returning an IA password only?